ui/components/multichain-accounts/multichain-account-menu/multichain-account-menu.tsx

```diff
@@ -28,14 +29,15 @@ export const MultichainAccountMenu = ({
   isRemovable,
   buttonBackgroundColor,
   handleAccountRenameAction,
+  isOpen = false,
+  onToggle,
```

Comment:
Updating this component to be a controlled component so we can ensure there is only one menu open at a time.